ext/web/00_infra.js

```diff
@@ -454,6 +455,13 @@ function pathFromURL(pathOrUrl) {
       ? pathFromURLWin32(pathOrUrl)
       : pathFromURLPosix(pathOrUrl);
   }
+
+  if (StringPrototypeStartsWith(pathOrUrl, "file:")) {
```

Comment:
Maybe it should check for `file:///` instead?

Reply:
We might have some compatibility issues with languages that don't include the "//" auth-path in the file URI to reduce it to `file:/path/to/file`, but given that most modern languages include it by default (JavaScript, Rust, C#, Java, Go) checking `file:///` should be good.

We can always change the check later if we find it too restrictive.